### PR TITLE
CONTRIBUTING.md and Repo Structure

### PR DESCRIPTION
**Pull request by bennyE31**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# So you want to contribute to pod-embedded-beta...
+Awesome! We're glad to have you!
+
+This set of guidelines will detail what you should know before contributing, as well as how to actually commit changes to the repository.
+
+## What do I need to know before I contribute?
+### Know where to get help!
+Always feel free to reach out on Slack, the #controls channel is a great place to start.
+### Get familiar with the repository!
+Take a look at the [readme](README.md) if you haven't already. It details the structure of the repository as well as how to set up your build environment.
```
Comment (EUdds) [inline CONTRIBUTING.md]: We should choose a convention for readme and stick with it. I.e readme/README/ReadMe

**Review by EUdds — CHANGES_REQUESTED** [review]
Looks good so far. Lets add some more stuff about contributing specifics. Like preferring OO, using readme documentation, etc.

File: CONTRIBUTING.md (hunk 1 above)
Comment (bennyE31) [reply CONTRIBUTING.md]: let's stick with all caps?

**Comment by bennyE31** [comment]
> Looks good so far. Lets add some more stuff about contributing specifics. Like preferring OO, using readme documentation, etc.

Pushed a commit with OO and readme doc details.

**Review by EUdds — APPROVED** [review]

**Comment by EUdds** [comment]
lgtm

**Pull request merged**